charts/pxc-database/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 appVersion: "1.0"
```

Comment:
appVersion should bump to 1.1.0 to match the operator version.

Reply:
ok